ckanapi/cli/main.py

```diff
@@ -92,7 +92,12 @@
 
 def parse_arguments():
     # docopt is awesome
-    return docopt(__doc__, version=__version__)
+    arguments = docopt(__doc__, version=__version__)
+    if not arguments['--config']:
+        arguments['--config'] = os.getenv('CKAN_INI', '')
+    if not arguments['--apikey']:
+        arguments['--apikey'] = os.getenv('CKANAPI_KEY', '')
+    return arguments
```

Comment:
it's not allowed to have both --config and --apikey so if either is passed the env vars should be ignored. when env vars are present let's prefer --config (alternatively we could exit with an error)

```suggestion
    arguments = docopt(__doc__, version=__version__)
    if not arguments['--config'] and not arguments['--apikey']:
        arguments['--config'] = os.getenv('CKAN_INI', '')
        if not arguments['--config']:
            arguments['--apikey'] = os.getenv('CKANAPI_KEY', '')
    return arguments
```